### DISCUSSION TITLE
Quick save/load masks

### DISCUSSION THREAD

**Discussion by alonsojr1980** [lllyasviel/Fooocus#2338] [Ideas]
First of all, thanks for Fooocus!!! If you could implement quick save/load methods of named masks, just like Photoshop, it would be really great! When inpainting for refining details, we usually have to do and redo the same masks a lot of times, the quick save/load would help with that! Thank you very much!

**Comment by mashb1t** [comment]
You could add a set of radio buttons below the mask with mask paths/presets and set the mask input value accordingly on radio button select.
This would need only a few gradio adjustments in the webui.py.
Hope this helps!
